content/applications/finance/accounting/customer_invoices/deferred_revenues.rst

```diff
@@ -2,194 +2,175 @@
+One entry, dated on the same day as the invoice, moves the invoice amounts from the income account
+to the deferred account. The other entries are deferral entries which, month after month, move
+the invoice amounts from the deferred account to the income account to recognize the revenue.
 
-Create a new entry
-------------------
+.. example::
+   You can defer a January invoice of $1200 over 12 months by specifying a start date of 01/01/2023
+   and an end date of 12/31/2023. At the end of August, $800 is recognized as an income,
+   whereas $400 remains on the deferred account.
 
-A **Deferred Revenues entry** automatically generates all journal entries in *draft mode*. They are
-then posted one by one at the right time until the full amount of the income is recognized.
+Reporting
+=========
 
-To create a new entry, go to :menuselection:`Accounting --> Accounting --> Deferred Revenues`, click
-on *Create*, and fill out the form.
+The deferred revenue report computes an overview of the necessary deferral entries for each account.
+To access it, go to :menuselection:`Accounting --> Reporting --> Deferred Revenue`.
 
-Click on **select related purchases** to link an existing journal item to this new entry. Some
-fields are then automatically filled out, and the journal item is now listed under the **Related
-Sales** tab.
+To view the journal items of each account, click on the account name and then :guilabel:`Journal
+Items`.
 
-.. image:: deferred_revenues/deferred_revenues05.png
-   :align: center
-   :alt: Deferred Revenue entry in Odoo Accounting
+.. image:: deferred_revenues/deferred_revenue_report.png
+   :alt: Deferred revenue report
 
-Once done, you can click on *Compute Revenue* (next to the *Confirm* button) to generate all the
-values of the **Revenue Board**. This board shows you all the entries that Odoo will post to
-recognize your revenue, and at which date.
-
-.. image:: deferred_revenues/deferred_revenues06.png
-   :align: center
-   :alt: Revenue Board in Odoo Accounting
+.. note::
+    Only invoices whose accounting date is before the end of the period of the report
+    are taken into account.
 
-What does "Prorata Temporis" mean?
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. _customer_invoices/deferred/generate_manually:
 
-The **Prorata Temporis** feature is useful to recognize your revenue the most accurately possible.
+Generate grouped deferral entries manually
+==========================================
 
-With this feature, the first entry on the Revenue Board is computed based on the time left between
-the *Prorata Date* and the *First Recognition Date* rather than the default amount of time between
-recognitions.
+If you have a lot of deferred revenues and wish to reduce the number of journal entries created, you
+can generate deferral entries manually. To do so, set the :guilabel:`Generate Entries` field in the
+**Settings** to :guilabel:`Manually & Grouped`. Odoo then aggregates the deferred amounts in a
+single entry.
```

Comment:
I used the contents of the tip that you had at the end of the file here instead.  I felt is was a) not really a tip and b) more useful here. I also removed the "example" part of the tip as I felt it wasn't necessary.